src/utils/themes.js

```diff
@@ -75,6 +77,8 @@ export const darkThemeColors = {
   smartWalletText: '#1D24D8',
   smartWalletSurface: '#f3f7ff',
   danger: '#ff0005',
+  labelTertiary: '#818eb3',
```

Comment:
We have this color as "accent" already.
;/ color naming is a bit messed up now, since colors in design files have been updated after color refactoring on front end